standard_releases.md

```diff
@@ -132,6 +112,37 @@ least work! The general process is:
   and in the `BODS-main` Transifex project, while the versioned release remains fixed.
 * Make the new version live on ReadTheDocs.
 
+#### Editing during a "freeze"
+
+During translation there may be a need to fix issues that exist on the `master`
+branch. If they do not affect anything that needs to be translated (docs,
+schema, codelists) they can be developed on a branch taken from `master` and
+merged back in when completed.
+  
+If the changes affect schema and/or docs then they may, in turn, affect the
+translation. The process for making these changes is:
+
+* First create an issue in Github to track the problem.
+* Then create a new branch that uses the issue number, e.g. `297-fix-typo` and
+  make the needed changes.
+* Determine if any of the strings you have changed have *already been
+  translated* in Transifex. If so, advise the translators of the nature of the
+  change you made to the English string. Even small changes, like fixing
+  spelling, punctuation or markup can cause Transifex to detect a new source
+  string and reset the translation. If the change is small, the translators
+  can reuse what they had before (with punctuation/markup fixes as applicable).
+  If the change is large (e.g. a complete sentence rewrite) they probably need
+  to translate it again from scratch. Agree when you are going to push the
+  updated string to Transifex, and proceed with the following steps at that time.
+* After review, merge the branch (`297-fix-typo`) into `master`.
+* Checkout the `master` branch and follow
+  [the translation process](https://github.com/openownership/data-standard/blob/master/README.md#managing-the-translation-workflow)
+  so that the new or changed strings are extracted, and push them to Transifex.
+* Wait for the new changes to be translated.
```

Comment:
Could we refer to the final instructions against "A new release" here? Is there any difference between pulling translated strings from an unedited Transifex project and an edited one?

Reply:
you're right, this is unnecessarily duplicative